README.md

```diff
@@ -57,14 +57,23 @@ Note that if you launch start.sh again after the installation, it will not downl
 
 ## Contribution tips
 
-- Add Transvision parent project as remote:
-"git remote add transvision git@github.com:mozfr/transvision.git"
+- Add Transvision parent project as a remote repository called 'upstream':
+```bash
+git remote add upstream git@github.com:mozfr/transvision.git
+```
 - Update your branch to the last version of Transvision:
```

Comment:
Same here, but explaining that it has to be done at least every time you start doing some development work.